src/material-experimental/mdc-chips/chip-grid.ts

```diff
@@ -60,12 +60,13 @@ export class MatChipGridChange {
 class MatChipGridBase extends MatChipSet {
   constructor(_elementRef: ElementRef,
               _changeDetectorRef: ChangeDetectorRef,
+              _dir: Directionality,
               public _defaultErrorStateMatcher: ErrorStateMatcher,
               public _parentForm: NgForm,
               public _parentFormGroup: FormGroupDirective,
               /** @docs-private */
               public ngControl: NgControl) {
-    super(_elementRef, _changeDetectorRef);
+    super(_elementRef, _changeDetectorRef, _dir);
```

Comment:
Has this change been captured in a separate PR already? This chips are used in Google, so this would be breaking otherwise

Reply:
That's just because the PR is currently based on Miles' MDC update PR. #16772.

<sup>Will add that to the PR description for now</sup>